packages/Table/index.js

```diff
@@ -1,11 +1,17 @@
 import React, { useState, useEffect } from 'react';
 import PropTypes from 'prop-types';
 import uuidv1 from 'uuid/v1';
-import Checkboxes from '@blaze-react/checkboxes';
+import orderBy from 'lodash.orderby';
+import Checkboxes from '../Checkboxes';
 
 const Table = ({ data: { columns, rows, identification }, onSelect, checkboxes, placeholder }) => {
+  const [_rows, setRows] = useState(rows);
   const [selected, setSelected] = useState([]);
```

Comment:
Usually when using the _ identifier on a variable it is to designate an internal value, like a parameter within a function or an internal function of an object.